lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -103,27 +119,27 @@ def write(project, target_installation_results)
         # @return [void]
         #
         def share_development_pod_schemes(project)
```

Comment:
can this become a class method too?

Reply:
I tried but this relies on a bit more of the state of the generator than I was looking to refactor in this PR. I might do another pass over this class in another PR now that it's fresh